dask/dataframe/io/tests/test_parquet.py

```diff
@@ -16,6 +17,9 @@
 from dask.dataframe.io.parquet.core import ParquetSubgraph
 from dask.utils import natural_sort_key, parse_bytes
 
+
+pytestmark = pytest.mark.skipif(da.numpy_compat._numpy_120, reason="Unsupported")
```

Comment:
We won't want this long-tem, since it's a blanket skip whenever we have NumPy 1.20. I've added it now just to keep the number of failures on this branch manageable.